lib/vapi/base.py

```diff
@@ -29,6 +29,7 @@
 from vapi.FileUserKeys import FileUserKeys
 from vapi.FileUserMetadata import FileUserMetadata
 from vapi.Categories import Categories
+from vapi.ConnectionDevices import ConnectionDevices
```

Comment:
I think that you need to add 
```
self.connection_devices = ConnectionDevices(self.server)
```
down in the line 74-89 range.

Reply:
Good catch!